src/parachain/amm/liquidity-pool/stable.ts

```diff
@@ -0,0 +1,280 @@
+import { CurrencyExt, StableLPToken } from "../../../types";
```

Comment:
Inspired by zenlink-dex package, logic is translated from their `sdk-core`

Reply:
Good job so far :)
And a big thank you.

Reply:
Thank you for the review!